src/pynwb/file.py

```diff
@@ -203,7 +203,7 @@ def __init__(self, **kwargs):
         pargs, pkwargs = fmt_docval_args(super(NWBFile, self).__init__, kwargs)
         pkwargs['name'] = 'root'
         super(NWBFile, self).__init__(*pargs, **pkwargs)
-        self.__start_time = datetime.utcnow()
+        self.__start_time = datetime.utcnow().isoformat() + "Z"
```

Comment:
According to the [schema documentation](http://pynwb.readthedocs.io/en/latest/pynwb.file.html?highlight=NWBFile) there's a `session_start_time` and a `file_create_date` but no `start_time` in `NWBFile`, so I don't know what this is. An accident perhaps? I think the value you meant to change is below.